newIDE/app/src/Utils/UseOpenInitialDialog.js

```diff
@@ -31,6 +35,16 @@ export const useOpenInitialDialog = ({
           openSubscriptionDialog({ reason: 'Landing dialog at opening' });
           removeRouteArguments(['initial-dialog']);
           break;
+        case 'signup':
+          const timeoutId = setTimeout(() => {
+            if (authenticated) {
+              openProfileDialog(true);
+            } else {
+              onCreateAccount();
+            }
+            removeRouteArguments(['initial-dialog']);
+          }, 2000);
+          return () => clearTimeout(timeoutId);
```

Comment:
Do you need to `break` from the `case` or is the return enough?

Reply:
The return is enough, otherwise eslint screams at you